lib/src/over_react_test/common_component_util.dart

```diff
@@ -249,15 +263,59 @@ void _testPropForwarding(BuilderOnlyUiFactory factory, dynamic childrenFactory()
     for (var forwardingTarget in forwardingTargets) {
       Map actualProps = getProps(forwardingTarget);
 
-      // If the forwarding target is a DOM element it will should not have invalid DOM props forwarded to it.
+      // If the forwarding target is a DOM element it should not have invalid DOM props forwarded to it.
       if (isDomElement(forwardingTarget)) {
         otherProps.forEach((key, value) {
-          expect(actualProps, isNot(containsPair(key, value)));
+          expect(actualProps[key], isNull, reason: unindent('''
```

Comment:
Why the change from containsPair to a null check?

Reply:
I mean... its been a month+... but IIRC, its because we're adding in `defaultProps` now - but I'd have to revert this and re-run common component tests in WSD to find the exact false negatives I was running into that gave me cause to update it.

Reply:
Changed back / fixed in https://github.com/Workiva/over_react_test/pull/102/commits/e8dbf1028a6b8421a391e41d7f912c8dfc785e8b